tools/bundletool/process_and_sign.sh.template

```diff
@@ -30,6 +30,40 @@ if [[ -n "$POST_PROCESSOR" ]]; then
   "$POST_PROCESSOR" "$WORK_DIR"
 fi
 
+# Strip out any unnecessary slices from embedded dynamic frameworks to save space
+for i in "$WORK_DIR"/*.app; do
+    # Lets figure out what slices the app has
+    app_name=$(basename $i .app)
```

Comment:
```suggestion
    app_name=$(basename "$i" .app)
```

Reply:
Yeah I'm slack with quotes... I will go and quote all the variables, they need it.